Docs/source/running_cpp/parameters.rst

```diff
@@ -1529,3 +1529,48 @@ WarpX supports checkpoints/restart via AMReX.
 * ``amr.restart`` (`string`)
     Name of the checkpoint file to restart from. Returns an error if the folder does not exist
     or if it is not properly formatted.
+
+Intervals parser
+--------------------------------------
+
+WarpX provides an interval parser that reads expressions in the input file of the form
+``1:2:3, 4: :,5:6,  :  ,etc.``.
+A comma is used as a separator between different slices and whitespaces are trimmed and not taken
+into account.
+A single slice can have 0, 1 or 2 colons.
```

Comment:
Let's keep it simple, especially with white spaces, for the first examples. Introduce one term at a time.
```suggestion
WarpX can parse interval expressions of the form ``start:stop:period``, e.g.
``1:2:3, 4::, 5:6, :, ::10``.
A comma is used as a separator between groups of intervals, which we call slices.
White spaces are ignored.
A single slice can have 0, 1 or 2 colons, just as `numpy slices <https://numpy.org/doc/stable/reference/generated/numpy.s_.html>`_, but with inclusive upper bound for ``stop``.
```